inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseIriPanel.java

```diff
@@ -155,7 +155,7 @@ public void onSelectionChanged(AjaxRequestTarget target, SchemaProfile bean)
         comboBox.setRequired(true);
         comboBox.add(Validators.IRI_VALIDATOR);
         comboBox.add(new LambdaAjaxFormComponentUpdatingBehavior("change", t -> {
-            // Do nothing just update the model values
+
```

Comment:
Please retain the comment. Empty code blocks should always be commented, otherwise it looks like a bug/missing code.